### ISSUE TITLE
Is there a way to follow AJAX queries?

### ISSUE THREAD

**Issue by yahyaerturan** [JCSama/CodeIgniter-develbar#6]
This is a fine library. I just wonder that if there is a way to follow AJAX queries as well?

**Comment by JCSama** [comment]
That's a new feature I'm working on it.

**Comment by JCSama** [comment]
@yahyaerturan  check the new version 0.8

**Comment by yahyaerturan** [comment]
Great! It catches the ajax calls and list them.

But there is no change in Database queries fyi. What I am targeting is to check the queries within ajax calls.

Another topic, last time and this time, I had to change `overflow:scroll` to `overflow:auto`. Otherwise I can't go over the tabs.

`.develbar ul li div.detail .scroll {overflow:auto}`

Meanwhile, I encountered an error with view.png file. It is resolved when I delete `data['icon']`. It is probably because I am using twig and there is no view data because of it.
